src/nvim/event/rstream.c

```diff
@@ -167,9 +169,16 @@ static void fread_idle_cb(uv_idle_t *handle)
   invoke_read_cb(stream, false);
 }
 
+static void read_event(void **argv)
+{
+  Stream *stream = argv[0];
+  bool eof = (uintptr_t)argv[1];
+  stream->read_cb(stream, stream->buffer, stream->data, eof);
```

Comment:
Does this also need a `NULL` check for `stream->read_cb`? There was an earlier `NULL` check in `invoke_read_cb()` when this was placed in the queue, but can we be sure that no one had used `rstream_start(...,NULL)` before `read_event()` is called?